packages/hash/api/src/model/knowledge/linkEntity.model.ts

```diff
@@ -0,0 +1,192 @@
+import {
+  GraphApi,
+  Filter,
+  EntityStructuralQuery,
+} from "@hashintel/hash-graph-client";
+import { Entity, Subgraph } from "@hashintel/hash-subgraph";
+import { getRootsAsEntities } from "@hashintel/hash-subgraph/src/stdlib/element/entity";
+
+import {
+  EntityMetadata,
+  EntityModel,
+  EntityProperties,
+  EntityTypeModel,
+  LinkEntityModel,
+} from "../index";
+
+/**
```

Comment:
Just commenting here for visibility, upstream PR was merged and we can address this now after updating branch

Reply:
Addressed in https://github.com/hashintel/hash/pull/1435/commits/b989e43090dec04e6da28fbe200c608b7fe7ad83